yarn-project/epoch-cache/src/epoch_cache.ts

```diff
@@ -296,4 +299,16 @@ export class EpochCache implements EpochCacheInterface {
     const committeeSet = new Set(committee.map(v => v.toString()));
     return validators.filter(v => committeeSet.has(v.toString()));
   }
+
+  async getRegisteredValidators(): Promise<EthAddress[]> {
+    if (
+      this.lastValidatorRefresh.getTime() + this.config.validatorRefreshIntervalSeconds * 1000 <
+      this.dateProvider.now()
+    ) {
+      const currentSet = await this.rollup.getAttesters();
+      this.allValidators = new Set(currentSet);
+      this.lastValidatorRefresh = new Date();
```

Comment:
```suggestion
      this.lastValidatorRefresh = this.dateProvider.now();
```

Reply:
Does epoch cache have a `dateProvider` now? I didn't think it did before.

Reply:
There is a dateProvider being used a few lines up

Reply:
Payback for this one :-P

https://github.com/AztecProtocol/aztec-packages/pull/15259#discussion_r2168939443